feat(dungeon): add realm clearing, misc, exalt options

sjkd23 opened the pull request:
- Added Realm Clearing and Exalt Dungeon as options for headcount/afkcheck

- Added the Speedy reaction emoji (for realm clearing HC)

- Updated the "dungeon" option shortcut for the logrun, headcount and afkcheck commands. Now includes ALL exalt dungeons and 'Misc' for miscellaneous dungeons.

ewang2002 commented on src/commands/logging/LogRun.ts:
```diff
@@ -68,10 +68,15 @@ export class LogRun extends BaseCommand {
                         stringChoices: [
                             { name: "o3", value: "ORYX_3" },
```
No idea why I never got notified about this, but yes -- make this a constant.

ewang2002 reviewed (changes requested):
As pat mentioned, make the O3 string choice a constant.

sjkd23 commented:
> As pat mentioned, make the O3 string choice a constant.

I dont currently know how to do this. But will attempt to figure it out. Probably.

sjkd23 commented:
Ok I think I did it right. It works with the test bot, and dmans is the one that walked me through it, so it should be good!

sjkd23 commented:
Added DUNGEON_SHORTCUTS as a type in types.ts 

Added Dungeon_Shortcuts as a constant in GeneralConstants

Exported dungeon shortcuts in logrun, headcount, and afkcheck commands,

ewang2002 reviewed (approved):
Nice refactoring.
